en/middleware/modules_main.md

```diff
@@ -1,20 +1,16 @@
-
 # Modules & Commands Reference
-The following pages document the PX4 modules, drivers and commands. They
-describe the provided functionality, high-level implementation overview and how
-to use the command-line interface.
 
-> **Note** **This is auto-generated from the source code** and contains the
-> most recent modules documentation.
+The following pages document the PX4 modules, drivers and commands. 
+They describe the provided functionality, high-level implementation overview and how to use the command-line interface.
+
+> **Note** **This is auto-generated from the source code** and contains the most recent modules documentation.
```

Comment:
FYI, main change here is to point to the docs API. I think that is useful.

Reply:
Agreed.